DOCSP-51889-retryable-reads-and-writes (#126)
mongodb/docs-kotlin-sync: source/crud/configure.txt

      :tabid: settings

      .. literalinclude:: /includes/configure-crud.kt
          :language: kotlin
          :dedent:
          :start-after: start-local-threshold-settings
          :end-before: end-local-threshold-settings

    .. tab:: Connection URI
      :tabid: uri

      .. literalinclude:: /includes/configure-crud.kt
          :language: kotlin
          :dedent:
          :start-after: start-local-threshold-uri
          :end-before: end-local-threshold-uri

In the preceding example, the {+driver-short+} distributes reads among matching members
within 35 milliseconds of the closest member's ping time.

Retryable Reads and Writes
--------------------------

> {+driver-short+} automatically retries certain read and write operations a single time

Reviewer: ```suggestion
The {+driver-short+} automatically retries certain read and write operations once
```

if they fail due to a network or server error.

You can explicitly disable retryable reads or retryable writes by setting the ``retryReads`` or
``retryWrites`` option to ``False`` in a ``MongoClientSettings`` instance. You can also
set the ``retryReads`` or ``retryWrites`` options in your connection URI.

> The following example sets both retryable reads and retryable writes to ``False``. Select the :guilabel:`MongoClientSettings` or :guilabel:`Connection URI`

Reviewer: ```suggestion
The following example sets both retryable reads and retryable writes to ``false``. Select the :guilabel:`MongoClientSettings` or :guilabel:`Connection URI`
```
suggestion: B/c it is lowercase in code
